src/Open3D/Geometry/LineSet.cpp

```diff
@@ -63,6 +63,16 @@ Eigen::Vector3d LineSet::GetMaxBound() const {
             });
 }
 
+Eigen::Vector3d LineSet::GetCenter() const {
+    Eigen::Vector3d center(0, 0, 0);
+    if (!HasPoints()) {
+        return center;
+    }
+    center = std::accumulate(points_.begin(), points_.end(), center);
```

Comment:
Same as before, Map the memory to Eigen matrice and use Eigen optimized functions